e2e/specs/customSchemaForm.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from "@playwright/test";
+import { FromBuilderPage } from "../pages";
+import customSchema from "../support/customSchema.json";
+
+test("Should be able to create a form using custom schema", async ({
+  page,
+}) => {
+  const fromBuilderPage = new FromBuilderPage(page);
+  await fromBuilderPage.gotoFormBuilder();
+
+  await page.getByRole("button", { name: "Create a new form" }).click();
```

Comment:
shall we move these elements to the page? 

ex: 

https://github.com/openmrs/openmrs-esm-patient-management/blob/d648f8110573591e605c1b9a4342f648ca54d5ee/e2e/pages/registrationAndEditPage.ts#L25

Reply:
sure @jayasanka-sack

Reply:
This comment is still valid @anjula-sack

Reply:
it is already moved @jayasanka-sack